### PR TITLE
Support swift model inference for llm pipeline

### PR DESCRIPTION
**Pull request by Firmament-cyou**

### CODE REVIEW

File: modelscope/pipelines/nlp/llm_pipeline.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # Copyright (c) Alibaba, Inc. and its affiliates.
 import os
 from contextlib import contextmanager
+from threading import Lock
 from typing import Any, Callable, Dict, Generator, Iterator, List, Tuple, Union
 
 import json
 import torch
+from swift import Swift
```
Comment (Jintao-Huang) [inline modelscope/pipelines/nlp/llm_pipeline.py:9]: 担心会循环依赖, swift不能import modelscope.pipelines. 我记一下好了

File: modelscope/pipelines/nlp/llm_pipeline.py [hunk 2]
```diff
@@ -27,6 +31,8 @@
 
 logger = get_logger()
 
+MODEL_ID_MAPPING = {v['model_id_or_path']: k for k, v in MODEL_MAPPING.items()}
```
Comment (wenmengzhou) [inline modelscope/pipelines/nlp/llm_pipeline.py:34]: MODEL_ID_MAPPING -> SWIFT_MODEL_ID_MAPPING

File: modelscope/pipelines/nlp/llm_pipeline.py [hunk 3]
```diff
@@ -170,6 +175,9 @@ def __init__(self,
             self.device_map = 'cuda'
         self.torch_dtype = kwargs.pop('torch_dtype', None)
         self.ignore_file_pattern = kwargs.pop('ignore_file_pattern', None)
+
+        if self._init_swift(kwargs['model'], kwargs.get('device', 'gpu')):
```
Comment (wenmengzhou) [inline modelscope/pipelines/nlp/llm_pipeline.py:179]: 这里根据framework=swift来确定是否要初始化swift? 而不是尝试初始化

File: modelscope/pipelines/nlp/llm_pipeline.py (hunk 2 above)
Comment (Firmament-cyou) [reply modelscope/pipelines/nlp/llm_pipeline.py]: done

File: modelscope/pipelines/builder.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Alibaba, Inc. and its affiliates.
```
Comment (wangxingjun778) [inline modelscope/pipelines/builder.py:1]: Please fix the unittest function case: tests.pipelines.test_human3d_render.Human3DRenderTest.test_run_modelhub

Delete: 'case_id': '3f2a7538253e42a8',
Add:  'case_id': '000039',

to pass the unittest run action.

**Pull request merged**